pkgs/top-level/aliases.nix

```diff
@@ -1419,7 +1419,7 @@ mapAliases ({
   pybitmessage = throw "pybitmessage was removed from nixpkgs as it is stuck on python2"; # Added 2022-01-01
   pyext = throw "pyext was removed because it does not support python 3.11, is upstream unmaintained and was unused"; # Added 2022-11-21
   pygmentex = texlive.bin.pygmentex; # Added 2019-12-15
-  pyload = throw "pyload has been removed from nixpkgs, as it was unmaintained"; # Added 2021-03-21
+  pyload = throw "pyload has been removed from nixpkgs. Use pyload-ng instead."; # Added 2021-03-21
```

Comment:
Update the date of this comment.

Reply:
Previously I got told that the old date should be kept :') https://github.com/NixOS/nixpkgs/pull/233100#discussion_r1206701931

I've now changed it to "Added 2021-03-21; Edited 2023-07-09"

Reply:
Better.